Lib/concurrent/futures/_base.py

```diff
@@ -193,9 +193,9 @@ def as_completed(fs, timeout=None):
 
     fs = set(fs)
     with _AcquireFutures(fs):
-        finished = set(
+        finished = {
```

Comment:
Let's skip all of these.  This module is delicate (harder to test minor changes) and the original code is marginally a bit clearer.